README.md

```diff
@@ -203,17 +203,20 @@ By default, latest images launch `prometheus` plugins to monitor fluentd. You ca
 
 ### Changelog
 **logzio/logzio-fluentd**:
+- v1.2.0:
+  - Upgrade plugin `fluent-plugin-kubernetes_metadata_filter` to `3.1.2`([#80](https://github.com/logzio/logzio-k8s/pull/80)).
```

Comment:
Can you add a comment regarding the fluentd as well?

Reply:
Done.